### DISCUSSION TITLE
Add support for sort command

### DISCUSSION THREAD

**Discussion by aklin2** [sandbox-bio/sandbox.bio#15] [Q&A] [answered]
The UNIX interface does not have the SORT command.
```
guest@sandbox$ sort
Program sort not found.
```
https://sandbox.bio/tutorials?id=bedtools-intro&step=8

Also, there is a small typo:
_"Many datasets of genomic features have many individual features that overlap one another (e.g. aligments from a ChiP seq experiment). It is often useful to just **cobine** the overlapping into a single, contiguous interval. The bedtools merge command will do this for you."_

**Comment by robertaboukhalil** [comment]
Nice catch, the typo will be fixed in the next release. We currently don't support most of the `coreutils` packages including `sort`, but that's on the list, I'll move this to the `Feature request` category. Thank you!

**Comment by robertaboukhalil** [comment, marked as answer]
@aklin2 `sort` is now supported! You may need to reset your cache.

Here's how to test it:

```
echo -e "10\n5\n15\n7" | sort -n
```

**Reply by aklin2** [reply]
Awesome! It works! Thanks!
